api/v1/db/json/pets-dao-example.js

```diff
@@ -46,4 +46,57 @@ const getPetById = id => new Promise((resolve, reject) => {
   }
 });
 
-module.exports = { getPets, getPetById };
+/**
+ * Posts a new pet
+ *
+ * @param {object} body Request body
+ * @returns {Promise} Promise object represents ?
+ */
+const postPet = body => new Promise((resolve, reject) => {
+  try {
+    const rawPets = readJsonFile(dbPath).pets;
+    const { name, owner, species } = body.data.attributes;
+
+    // Determine smallest unused id
+    const ids = _.mapValues(rawPets, 'id');
+    const idHash = {};
+    _.forEach(ids, (id) => {
+      idHash[id] = true;
+    });
+    let id = 1;
+    while (idHash[id]) {
+      id += 1;
+    }
+    rawPets.push({
+      id, name, owner, species,
```

Comment:
Could you break these properties over multiple lines like in our [style guide](https://github.com/osu-mist/style-guides/blob/master/JavaScript.md#properties-spanning-multiple-lines)?